### DISCUSSION TITLE
I have an issue in roll back of version in  power apps

### DISCUSSION THREAD

**Discussion by Vinaymhn27** [community/community#132356] [New to GitHub]
Hi,

can someone help me in rolling back the version in power apps . As i see a different version in published version while editing i see a different version.
kindly help me in solving this issue as its impacting our development deadlines.

**Comment by rishiTamoghna17** [comment]
Check Published Version: First, confirm which version is currently published and verify its functionality.

Review Version History: In Power Apps, you can review the version history to see when changes were made and which versions were published. This helps in identifying discrepancies between what's published and what's in the editor.

Rollback to Previous Version:

Navigate to the version history or versions panel in Power Apps.
Identify the last known stable version or the version you want to revert to.
Select that version and choose the option to publish it. This will replace the current published version with the selected one.
Publish Changes: Once you've selected the correct version to revert to, publish it to ensure that the published version matches what you see in the editor.

Verify Functionality: After publishing, thoroughly test the application to ensure that everything is functioning as expected in the published version.

**Reply by Vinaymhn27** [reply]
Hi @rishiTamoghna17 , Thanks for the response...But the problem here is that published version is selected properly and published app is also correct but when we click to edit the canvas app its totally different version . The above steps were tried already and we dont see any changes there.
